app/Filament/Resources/Shop/ProductResource.php

```diff
@@ -213,6 +213,10 @@ public static function getFormSchema(string $layout = Forms\Components\Grid::cla
     public static function getTableColumns(): array
     {
         return [
+            Tables\Columns\ImageColumn::make('media')
```

Comment:
We have a SpatieMediaLibraryImageColumn right?

Reply:
Thanks. Didn't see that one.